docs/data/toolpad/getting-started/why-toolpad.md

```diff
@@ -0,0 +1,72 @@
+- As backends evolve, the internal applications need to evolve with them with minimal amount of maintenance required.
+- Lesser focus on quality assurance usually leads to higher chances of failure.
+
+## TL;DR
+
+- Integrates well with your existing code. You can reuse your database models, client libraries, secrets, bespoke components
+- Integrates well with your development lifecycle. Your project lives on your own file system. You can use your own IDE, version control system, deployment target, CLI tools
+- Comes with a drag-and-drop builder to simplify UI building
+- Sets healthy UI constraints that prevent common pitfalls and anti-patterns
+- Lets you maintain a single source of truth for all business logic
+- Less boilerplate so you can focus on the important parts of the app
+
+## Market overview
+
+After reviewing multiple approaches and tools, we noticed that there are a few recurring core problem areas that needed to be addressed:
+
+### The challenge of codebase fragmentation and duplication
+
+Frequently, developers invest valuable time in hunting down particular versions of business logic and then maintaining them at multiple places. When working with external tools, code duplication can often arise as consequences of design decisions. Not having a unified reference point within a codebase compromises its scalability, ease of maintenance and debugging.
+
+### Loss of control with external tools
+
+When working with external tools, developers often forfeit their autonomy and are constrained by the choices offered by the tool. Developers typically derive a sense of ownership and responsibility when they oversee the entire process, encompassing integrations, deployment, and version control. However, when a black-box element intervenes, control often shifts away.
+
+### Trade-off between speed and customization
+
+Typically, depending on factors such as project size, time constraints, available resources, and scope, developers are compelled to strike a balance between speed and customization. Every business wants features to be delivered at speed, but being able to customize later keeps the project open to come what may. While both aspects are significant, we believe that an architecturally well-designed tool empowers you to attain both objectives simultaneously.
+
+### Trust and Safety
+
+Internal tools have access to highly sensitive data, data that every organization solemnly pledges to safeguard. Despite the stringent security measures in place, data breaches remain a prevalent occurrence. Organizations rigorously engage in due diligence and compliance assessments before placing trust in a vendor. The security of the underlying data remains an exceptionally vital aspect that cannot be underestimated.
+
+## Toolpad
+
+We think addressing the above mentioned problems can lead to a better developer experience. Toolpad solves them in the following ways:
```

Comment:
No need to speculate about what you "think"—say it like you mean it! It's true if we say that it is. 😁

```suggestion
Addressing all the aforementioned problems leads to a better developer experience. Toolpad solves them in the following ways:
```